ambry-rest/src/main/java/com/github/ambry/rest/NettyRequest.java

```diff
@@ -661,4 +661,32 @@ public void invokeCallback(Exception exception) {
       }
     }
   }
+
+  /**
+   * The {@link NettyRequest}'s implementation of {@link com.github.ambry.rest.RestRequest.RestRequestContext}. It carries
+   * {@link Channel} and {@link HttpRequest}.
```

Comment:
The comment says it carries both `Channel` and `HttpRequest`, could you tell me why it only keeps `Channel` inside and no `HttpRequest`?

Reply:
sorry, i already removed the http request, will update the comments